test/plain-list/AutosuggestApp.test.js

```diff
@@ -406,6 +730,29 @@ describe('Plain list Autosuggest', () => {
     });
   });
 
+  describe('inputProps.onChange with selectFirstSuggestion', () => {
+    before(() => {
+      setSelectFirstSuggestion(true);
+    });
+
+    after(() => {
+      setSelectFirstSuggestion(false);
+    });
+
+    beforeEach(() => {
+      onChange.reset();
+    });
+
+    it('should be called with the right parameters when user types', () => {
+      focusAndSetInputValue('c');
+      expect(onChange).to.have.been.calledOnce;
+      expect(onChange).to.be.calledWith(eventInstance, {
+        newValue: 'c',
+        method: 'type'
+      });
+    });
+  });
```

Comment:
Here `setSelectFirstSuggestion` is set to true. This is testing that `onChange` is only called once -- i.e. that `onChange` was not called when auto-selecting the first suggestion.